types/chrome-remote-interface/index.d.ts

```diff
@@ -1,6 +1,6 @@
 // Type definitions for chrome-remote-interface 0.30
 // Project: https://github.com/cyrus-and/chrome-remote-interface
-// Definitions by: Khairul Azhar Kasmiran <https://github.com/kazarmy>
+// Definitions by: Khairul Azhar Kasmiran <https://github.com/kazarmy>, Seth Westphal <https://github.com/westy92>
```

Comment:
```suggestion
// Definitions by: Khairul Azhar Kasmiran <https://github.com/kazarmy>
//                 Seth Westphal <https://github.com/westy92>
```
it's also valid for parser and more readable